openmetadata-ui/src/main/resources/ui/src/components/searched-data/SearchedData.tsx

```diff
@@ -160,8 +160,8 @@ const SearchedData: React.FC<SearchedDataProps> = ({
                         hideOnSinglePage
                         className="text-center"
                         current={
-                          isNumber(Number(page ?? currentPage))
-                            ? Number(page ?? currentPage)
+                          isNumber(Number(currentPage ?? page))
```

Comment:
Earlier we have preceded page over currentPage, now we have changed it with currentPage
what was the issue & does it affect at other places?

Reply:
@chirag-madlani  have removed the `currentPage` props and got it accessible through the page variable. Have changed the pagination method to the URL method.